docs/BR.md

```diff
@@ -1756,142 +1754,916 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+| Any other value                                                                    | SHOULD NOT   | -                       | -                     | -                              |
+
+#### 7.1.2.5 TLS Subordinate CA Certificate Profile
+
+| __Field__                      | __Description__ |
+| ---                            | ------          |
+| `tbsCertificate`               | |
+| \ \ \ \ `version`              | MUST be v3(2) |
+| \ \ \ \ `serialNumber`         | MUST be a non-sequential number greater than zero (0) containing at least 64 bits of output from a CSPRNG. |
+| \ \ \ \ `signature`            | See [Section 7.1.3.2](#7132-signature-algorithmidentifier) |
+| \ \ \ \ `issuer`               | MUST be byte-for-byte identical to the `subject` field of the Issuing CA. See [Section 7.1.4.1](#7141-name-encoding) |
+| \ \ \ \ `validity`             | __**TBD**__ |
+| \ \ \ \ `subject`              | See [Section 7.1.2.8.1](#71281-ca-certificate-naming) |
+| \ \ \ \ `subjectPublicKeyInfo` | See [Section 7.1.3.1](#7131-subjectpublickeyinfo) |
+| \ \ \ \ `issuerUniqueID`       | MUST NOT be present |
+| \ \ \ \ `subjectUniqueID`      | MUST NOT be present |
+| \ \ \ \ `extensions`           | See [Section 7.1.2.5.1](#71251-tls-subordinate-ca-extensions) |
+| `signatureAlgorithm`           | Encoded value MUST be byte-for-byte identical to the `tbsCertificate.signature`. |
+| `signature`                    | |
+
+##### 7.1.2.5.1 TLS Subordinate CA Extensions
+
+| __Extension__                     | __Presence__  | __Critical__          | __Description__ |
+| ----                              | -             | -                     | ----- |
+| `authorityKeyIdentifier`          | MUST          | N                     | See [Section 7.1.2.9.1](#71291-authority-key-identifier) |
+| `basicConstraints`                | MUST          | Y                     | See [Section 7.1.2.8.3](#71283-basic-constraints) |
+| `certificatePolicies`             | MUST          | N                     | See [Section 7.1.2.8.4](#71284-certificate-policies---affiliated-ca) (Affiliated CA) |
+| `crlDistributionPoints`           | MUST          | N                     | See [Section 7.1.2.9.2](#71292-crl-distribution-points) |
+| `keyUsage`                        | MUST          | Y                     | See [Section 7.1.2.8.8](#71288-key-usage) |
+| `subjectKeyIdentifier`            | MUST          | N                     | See [Section 7.1.2.9.4](#71294-subject-key-identifier) |
+| `extKeyUsage`                     | MUST[^eku_ca] | N                     | See [Section 7.1.2.8.7](#71287-extended-key-usage---tls-ca) (TLS CA) |
+| `nameConstraints`                 | MUST          | \*[^name_constraints] | See [Section 7.1.2.8.9](#71289-name-constraints) |
```

Comment:
Name constraints for normal TLS Subordinate CAs should be prohibited (or else it's a technically constrained CA as profiled in a prior section).  Change to "MUST NOT be present"

Reply:
Sorry, this was meant to be a MAY, not a MUST

NameConstraints for a normal TLS subordinate CA are fine, but it's not a technically constrained sub-CA unless it meets the profile. For example, if you only constrain by DNS, it's not technically constrained, but it's also not misissuance.

This is allowed under the current BRs. Are you proposing that we change that, such that the only nameConstraints permitted are technically constrained? That's not unreasonable (to avoid confusion), but just wanting to make sure that was explicit if so.

Reply:
I think MAY is better.  You can have Name constrained CAs that are not technically constrained as you pointed out.